recipes/openmpi/all/test_package/conanfile.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class OpenmpiTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is
+        # in "test_package"
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
+        self.copy("*.dll", dst="bin", src="bin")
+        self.copy("*.dylib*", dst="bin", src="lib")
+        self.copy("*.so*", dst="bin", src="lib")
+
+    def test(self):
+        if not tools.cross_building(self):
+            os.chdir("bin")
+            self.run(".%sexample" % os.sep)
```

Comment:
```suggestion
            bin_path = os.path.join("bin", "example")
            self.run(bin_path, run_environment=True)
```

Reply:
Why is this change needed? I'll do it but I'd like to understand the reason :)

Reply:
Better comprehension and `run_environment=True` will add “lib” and “bin” folders of the dependencies to the `PATH`, `LD_LIBRARY_PATH` and `DYLIB_LIBRARY_PATH` env variables, so you don't need to import dll, dylib and so.